library/src/main/java/com/anjlab/android/iab/v3/BillingProcessor.java

```diff
@@ -610,6 +615,7 @@ private boolean purchase(Activity activity, List<String> oldProductIds, String p
 							purchaseType,
 							purchasePayload);
 				}
+				//todo check if null check should be removed or if else should be removed
```

Comment:
I take it you mean that the null check if unnecessary. Resolved here 329c287af6d4f6f3c5d422051fc420279e29b29d